apollo-compiler/src/test/graphql/com/example/inline_fragments_with_friends/QueryExpected.java

```diff
@@ -0,0 +1,48 @@
+package com.example.inline_fragments_with_friends;
+
+import java.lang.Float;
+import java.lang.String;
+import java.util.List;
+import javax.annotation.Nullable;
+
+public interface Query {
+  @Nullable Hero hero();
+
+  interface Hero {
+    String name();
+
+    @Nullable
+    AsHuman asHuman();
+
+    @Nullable
+    AsDroid asDroid();
```

Comment:
I wonder how this annotation in a separate line didnt fail your test. All other annotations are on the same line